### ISSUE TITLE
Getting Error when trying to clean

### ISSUE THREAD

**Issue by jam1015** [k-takata/minpac#144]
I call

```
:call minpac#clean()
```
and I get this error:

```
Removing the above directory. [y/N]? y
Error detected while processing function minpac#clean[2]..minpac#impl#clean:
line   41:
E944: Reverse range in character class
Press ENTER or type command to continue
```
If it even comes up.  Sometimes nothing happens.  
Also I can't see the above directory; nothing is listed, makes me think the display is not popping up correctly.

**Comment by jam1015** [comment]
Fixed it somehow by manually deleting all my plugins, and then reinstalling them.  It worked after that.  Still curious what the problem was though.

**Comment by k-takata** [comment]
I think that this was a bug of Vim and should have been fixed in [v8.1.1378](https://github.com/vim/vim/commit/701ff0a3e53d253d7300c385e582659bbff7860d).
